queue.c

```diff
@@ -369,8 +375,7 @@ BaseType_t xQueueGenericReset( QueueHandle_t xQueue,
         }
         else
         {
-            traceQUEUE_CREATE_FAILED( ucQueueType );
-            mtCOVERAGE_TEST_MARKER();

```

Comment:
Why was this removed? It seems that it is there for logging/tracing. Or am I missing something?

Reply:
This case of `pxQueue` being `NULL` cannot be reached, and the trace event seems to belong to the dynamic case.